whats_new/terminology.adoc

```diff
@@ -57,4 +57,4 @@ link:../architecture/core_concepts/pods_and_services.html#pods[pods].
 link:../architecture/infrastructure_components/kubernetes_infrastructure.html#master[_Masters_]
 in OpenShift v3 do the job of the _broker_ layer in OpenShift v2. However, the
 MongoDB and ActiveMQ layers used by the broker in OpenShift v2 are no longer
-necessary because [sysitem]#etcd# is typically installed with each master.
+necessary because the link:../architecture/infrastructure_components/kubernetes_infrastructure.html#master[key-value database] [sysitem]#etcd# is typically installed with each master.
```

Comment:
You can update this old timey `[sysitem]` usage to some current markup choice (per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference). I've been using `*` for etcd as it seemed to best fit as a "System term, daemon, service, or software package".